packages/istanbul-api/lib/reporter.js

```diff
@@ -88,6 +97,32 @@ Reporter.prototype = {
                 )
         );
 
+        // TODO:
```

Comment:
nyc has its own implementation too.. and that doesn't allow configuration
so also.. change both?